.github/workflows/docs.yml

```diff
@@ -0,0 +1,35 @@
+name: Docs
+on: [push, release]
+
+jobs:
+  notebooks:
+    name: "Build the notebooks for the docs"
+    runs-on: ubuntu-latest
```

Comment:
You may want to pick a fixed `ubuntu` version to make build more stable, as new versions may easily break the build.